addons/mail/static/src/models/message/message.js

```diff
@@ -779,6 +793,14 @@ function factory(dependencies) {
             default: "",
             dependencies: ['body'],
         }),
+        messageTypePrefix: attr({
```

Comment:
guideline: fields should be sorted alphabetically

comment applies to the whole diff

Reply:
Done. Most fields were removed anyway